docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -138,52 +96,9 @@ If you want to build the PyBaMM Docker image locally from the PyBaMM source code
 
       conda activate pybamm
 
-Building Docker images with optional arguments
-----------------------------------------------
-
-When building the PyBaMM Docker images locally, you have the option to include specific solvers by using optional arguments. These solvers include:
-
-- ``IDAKLU``: For IDA solver provided by the SUNDIALS plus KLU.
-- ``JAX``: For Jax solver.
-- ``ALL``: For all the above solvers.
-
-To build the Docker images with optional arguments, you can follow these steps for each solver:
-
-.. tab:: JAX solver
-
-      .. code-block:: bash
-
-            docker build -t pybamm:jax -f scripts/Dockerfile --build-arg JAX=true .
-
-.. tab:: IDAKLU solver
-
-      .. code-block:: bash
-
-            docker build -t pybamm:idaklu -f scripts/Dockerfile --build-arg IDAKLU=true .
-
-.. tab:: All solvers
-
-      .. code-block:: bash
-
-            docker build -t pybamm:all -f scripts/Dockerfile --build-arg ALL=true .
-
-After building the Docker images with the desired solvers, use the ``docker run`` command followed by the desired image name. For example, to run a container from the image built with all optional solvers:
-
-.. code-block:: bash
-
-      docker run -it pybamm:all
-
-Activate PyBaMM development environment inside docker container using:
-
-.. code-block:: bash
-
-      conda activate pybamm
-
-If you want to exit the Docker container's shell, you can simply type:
-
-.. code-block:: bash
+.. note::
 
-      exit
+  PyBaMM Docker image comes with all the solvers by default. These solvers include ``IDAKLU`` IDA solver provided by the SUNDIALS plus KLU and ``JAX`` solver.
```

Comment:
```suggestion
  PyBaMM's Docker image comes with all available solvers by default. These solvers include ``IDAKLU`` IDAS solver provided by the SUNDIALS linked with SuiteSparse's KLU and the ``JAX`` solver.
```

A small comment on the wording here. The IDAKLU solver is the IDA solver with sensitivities, hence the extra S ([`IDAS`](https://sundials.readthedocs.io/en/latest/idas/)). SUNDIALS is not shipped with KLU by default and is provided by SuiteSparse.